subprojects/kotlin-dsl/build.gradle.kts

```diff
@@ -49,6 +54,12 @@ dependencies {
     implementation(libs.guava)
     implementation(libs.inject)
 
+    implementation(libs.futureKotlin("script-runtime")) {
+        isTransitive = false
+    }
+    implementation(libs.futureKotlin("daemon-embeddable")) {
+        isTransitive = false
+    }
```

Comment:
Why the two `isTransitive = false`? I t does not look right conceptually and I don't think it has any effect as both things are fully dependencies further down (in `kotlin-compiler-embeddable/build.gradle.kts `).

Reply:
You're right, it doesn't make sense, fixed, thanks!